.vscode/settings.json

```
@@ -3,7 +3,7 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.format": true
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.format": "explicit"
```

```suggestion
    "source.fixAll.eslint": "true",
    "source.fixAll.format": "true"
```

I'll revert this. Waiting for CI to deploy.